open_close.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+
+vent_status = input('enter the vent status')
+
+
+def open_or_close(vent_status):
+    is_input_valid(vent_status)
+    wanted_status = get_wanted_vent_status()
+    if vent_status == wanted_status:
+        print('you dont need to change the status')
+    else:
+        print('you need to change the status')
+
+
+def is_input_valid(situation):
+    if situation == 'open' or situation == 'close':
+        return
+    else:
+        print('The inserted situation is not accepted')
+        exit()
+
+
+def get_wanted_vent_status():
+    current = datetime.datetime.now().time()
+    start = datetime.time(12, 0, 0)
+    end = datetime.time(16, 0, 0)
+    time = start <= current <= end
+    if time == True:
+        wanted_status = 'close'
+        return wanted_status
+    start = datetime.time(16, 0, 0)
+    end = datetime.time(21, 0, 0)
+    time = start <= current <= end
+    if time == True:
+        wanted_status = 'open'
+        return wanted_status
+
+    else:
```

Comment:
You don't need the else. 
You can just return the `open` status, since if we got this fur, this status needs to be returned anyway.

You can replace:
```python
else:
    wanted_status = 'open'
    return wanted_status
```

With:
```python
return 'open'
```